assets/.npmrc

```diff
@@ -0,0 +1 @@
+save-exact=true
```

Comment:
i think we should get into the habit of pinning deps even though we're using a lockfile